### PR TITLE
Kyma release 1.17.0-rc2

### PR DESCRIPTION
**Pull request by adamwalach**
**Description**

Changes proposed in this pull request:

- installer bumped to 1.17.0-rc2
- ...
- ...

**Related issue(s)**
https://github.com/kyma-project/kyma/issues/9833

### CODE REVIEW

File: resources/uaa-activator/templates/job.yaml [hunk 1]
```diff
@@ -24,7 +24,7 @@ spec:
       serviceAccountName: {{ include "uaa-activator.fullname" . }}
       initContainers:
```
Comment (jasiu001) [inline resources/uaa-activator/templates/job.yaml:25]: `initContainers` is not needed anymore, should be removed

File: resources/uaa-activator/values.yaml [hunk 2]
```diff
@@ -6,11 +6,14 @@ global:
...
+    developer: KymaRuntimeDeveloper
+    namespaceAdmin: KymaRuntimeNamespaceAdmin
 
 image:
   registryPath: eu.gcr.io/kyma-project
   pullPolicy: IfNotPresent
-  version: "a0101415"
+  version: "94ac5eff"
 
 initJobs:
```
Comment (jasiu001) [inline resources/uaa-activator/values.yaml:18]: after removing initContainer, all `initJobs` keys are not needed

**Review by jasiu001 — DISMISSED** [review]

**Review by jasiu001 — APPROVED** [review]

**Review by crabtree — APPROVED** [review]

**Review by strekm — APPROVED** [review]
/meow

**Review by franpog859 — APPROVED** [review]

**Review by clebs — APPROVED** [review]

**Review by nachtmaar — APPROVED** [review]

**Pull request merged**